test/selenium/lib/view/signed_in/view.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var webdriver = require('selenium-webdriver');
+var SignedInPageAccessor = require('./accessors.js');
+var SignedOutPageView = require('../signed_out/view.js');
+var signedOutPageView;
+
+function SignedInPageView(driver) {
+    this.driver = driver;
+    this.accessors = new SignedInPageAccessor(this.driver);
+    signedOutPageView = new SignedOutPageView(this.driver);
```

Comment:
I'd put the the initialization of the view directly in functions like `signOut`. However, if we put it here. there might be an advantage that I didn't see. Is there an advantage of putting it here?